web-test-runner.config.js

```diff
@@ -74,20 +78,24 @@ export default {
           'lcov',
           {
             start() {
-              // eslint-disable-next-line no-console
-              console.log(
-                `Code coverage report: ${chalk.bgBlue(
-                  'http://localhost:8080',
-                )}\n`,
-              );
+              if (process.env.NODE_ENV === 'development') {
```

Comment:
We don't serve the test coverage report with production builds. So no need to show a link to it.